contracts/tropykus/TropykusLendingService.sol

```diff
@@ -5,19 +5,28 @@ import "../services/LendingService.sol";
 import "../smartwallet/SmartWalletFactory.sol";
 import "../smartwallet/SmartWallet.sol";
 import "../smartwallet/IForwarder.sol";
+import {IPriceOracleProxy, IComptrollerG6, IcErc20} from "contracts/tropykus/ITropykus.sol";
+import "./TropykusCommon.sol";
 
-contract TropykusLendingService is LendingService {
-    address private _crbtc;
+contract TropykusLendingService is LendingService, TropykusCommon {
     SmartWalletFactory private _smartWalletFactory;
     uint256 private constant _UNIT_DECIMAL_PRECISION = 1e18;
+    address private _comptroller;
+    address private _crbtc;
+
+    struct TropykusContracts {
+        address comptroller;
+        address crbtc;
+    }
```

Comment:
I think this struct could be moved to `TropykusCommon` since `TropykusBorrowService` probably will need it too.

Reply:
There is one in each contract since the lending contract doesn't need all of the properties, but now that you mention this I think its better to share the struct and just omit the properties that aren't needed. Thanks!